application/src/utils/config.py

```diff
@@ -1,7 +1,10 @@
+# problem with importing
+#  from ..gui.modules.settings import width, height, font, opaci, frame, top
 import os
 import sys
 import toml
 
+# i want to connect variables from settings to config file and make it update every time any of variables change
 CFG_VERSION = '0.7'
```

Comment:
yeah, like I wrote in first comment.

I think it should be other way around. New class should be introduced and whole config management should be dome via it's instance.

Reply:
Yea, I will try with this.